embrace-android-sdk/src/androidTest/assets/golden-files/session-end.json

```diff
@@ -147,7 +141,10 @@
       "attributes": {
         "emb.sequence_id": "1",
         "emb.private": "true",
-        "emb.type": "SESSION"
+        "emb.type": "SESSION",
+        "emb.okhttp3": "true",
```

Comment:
```suggestion
        "emb.okhttp3_on_classpath": "true",
```

Reply:
Sounds good. I'm I'm got tweak the implementation a bit given the VERSION file was actually moved in 4.7, so I'll make this change along with changing that